### DISCUSSION TITLE
How to install turso CLI in windows 8.1

### DISCUSSION THREAD

**Discussion by RProduction** [tursodatabase/turso-cli#676] [Q&A]
Hello, I want to use turso, but seem turso cli need linux subsystem.
Is there is a way to install turso CLI in my OS?
Do I need to build the cli from scratch using go?

**Comment by haaawk** [comment]
Right now the only option would be to build from source. Sorry about that.

**Reply by RProduction** [reply]
I see, yeah I tried it.
Seem it need go latest version.
I am using go v20 because that the latest version that support win 8.1.
For now I am using another database.
It will be great if turso CLI installation can be easily initiated using npm.

**Reply by martinmiglio** [reply]
Are there any plans for a native windows CLI to be added in the future? Using WSL is really cramping my style.

**Reply by haaawk** [reply]
Not in the near future. Hopefully a web ui would cover your needs -> https://turso.tech/app
